addons/difficulty/fnc_unitInit.sqf

```diff
@@ -35,7 +35,8 @@ if (
 };
 
 // Init unit
-if ( GVAR(enabled) && {_unit getVariable [QGVAR(enabled), true]}) then {
+if (
+	GVAR(enabled) && {_unit getVariable [QGVAR(enabled), true]} && {!(_unit getVariable [QGVAR(initialized), false])}) then {
```

Comment:
Why double negative here? Wouldn't _unit getVariable [QGVAR(initialized), true] make more sense?

Reply:
It indicates the default value, if it isn't set.

Reply:
Oh I see my bad.